geometry_msgs/Pose2D ally1
geometry_msgs/Pose2D ally2
geometry_msgs/Pose2D opp1
geometry_msgs/Pose2D opp2
geometry_msgs/Pose2D ball